### PR TITLE
Pull Request/Issue Templates Update (20210923)

### PR DESCRIPTION
**Pull request by MingcongBai**
Topic Description
-----------------

Update issue template to reflect workflow changes discussed in Minutes 2021-03-29 and on AOSCC 2021 Day 2. For details regarding these changes, please refer to the [AOSC OS Topic-Based Maintenance Guidelines](https://wiki.aosc.io/developer/packaging/topic-based-maintenance-guideline/).

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE [hunk 1]
```diff
@@ -13,23 +13,25 @@ Package(s) Affected
 Security Update?
 ----------------
 
-<!-- If this topic is part of a security update, please uncomment "Yes,"
-     and mark with the `security` label, as well as reference issue number below for priority processing. -->
+<!-- If this topic contains security update(s), please uncomment "Yes,"
+     and mark with the `security` and `priority` labels, as well as reference issue number below for prioritised processing. -->
```
Comment (KexyBiscuit) [inline .github/PULL_REQUEST_TEMPLATE]: Are there other situations requiring the priority label?

File: .github/PULL_REQUEST_TEMPLATE [hunk 2]
```diff
@@ -13,23 +13,25 @@ Package(s) Affected
...
 
 <!-- Yes - Issue Number: ISSUENUMBER -->
```
Comment (KexyBiscuit) [inline .github/PULL_REQUEST_TEMPLATE:23]: I prefer to use [Linking a pull request to an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) feature instead.

File: .github/PULL_REQUEST_TEMPLATE (hunk 1 above)
Comment (MingcongBai) [reply .github/PULL_REQUEST_TEMPLATE]: We'll see. For now, this is a description for the security field.

File: .github/PULL_REQUEST_TEMPLATE (hunk 2 above)
Comment (MingcongBai) [reply .github/PULL_REQUEST_TEMPLATE:23]: Got it.

**Review by KexyBiscuit — APPROVED** [review]

**Pull request merged**